internal/cli/runner_profile_delete.go

```diff
@@ -1,17 +1,26 @@
 package cli
 
 import (
+	"github.com/posener/complete"
```

Comment:
It was recently suggested to me to not re-order the import statements, so I'm passing that along here! 😄

Reply:
Ha! We talked about this once. My memory was that we decided we have three groupings:

```
stdlib

external packages

internal packages
```

I'll put this on the team discussion list.